src/app/server/AppDelegate.h

```diff
@@ -22,12 +22,18 @@
 
 #pragma once
 
+#include <lib/core/CHIPError.h>
+
 class AppDelegate
 {
 public:
     virtual ~AppDelegate() {}
+    /**
+     * This is called on start of session establishment process
```

Comment:
This says nothing useful.  What is considered the "start of the session establishment process" @sharadb-amazon?  This specifically gets called when PBKDFParamRequest is received, right?

Reply:
Going to address it in this PR: https://github.com/project-chip/connectedhomeip/pull/28186